glideinwms/sources/factory_entries.py

```diff
@@ -74,6 +75,12 @@ def acquire(self):
                     nretries=self.nretries,
                     retry_interval=self.retry_interval)
 
+                if correction_map is not None:
```

Comment:
Tested this code in-situ on hepcsvc03 and it worked as scheduled.  I leave the task of determining code acceptability to others but functionally it is just what I requested and needed.